be2-scala/src/main/scala/ch/epfl/pop/model/objects/LaoData.scala

```diff
@@ -36,8 +36,8 @@ final case class LaoData(
   def updateWith(message: Message, adr: Option[String]): LaoData = {
     message.decodedData.fold(this) {
       case call: CloseRollCall => adr match {
-        case Some(str) => LaoData(owner, call.attendees, privateKey, publicKey, witnesses, str)
-        case _ => LaoData(owner, call.attendees, privateKey, publicKey, witnesses, address)
+        case Some(str) => LaoData(owner, owner :: call.attendees, privateKey, publicKey, witnesses, str)
+        case _ => LaoData(owner, owner :: call.attendees, privateKey, publicKey, witnesses, address)
```

Comment:
so why did that change? It makes sense that the owner is an attendee, (although I think front ends do it anyway) but what does this have to do with the bug?

Reply:
I mentioned it on Slack, but briefly, the back end was always throwing an error after trying to close a re opened RollCall: "invalid keys attendee", because, in the updateWith function of LaoData, we did not include the PoP token of the LAO (aka the owner). With a first close RollCall (after opening it for the first time), the error did not occur, since the LaoData was correctly already included the owner in the attendees field (done during CreateLao). 
Well, it seems as the `attendees` field in a RollCall message coming from the front end do not include the owner (PoP token of LAO), but only the generated PublicKey by the RollCall. Therefore, I decided to add the owner into the attendees List after a rollcall close process.

Reply:
Oh yeah that part is just below. That makes perfect sense, thank you!